test/tools/libtesteth/TestOutputHelper.cpp

```diff
@@ -124,14 +124,49 @@ bool pathHasTests(boost::filesystem::path const& _path)
     return false;
 }
 
-void TestOutputHelper::checkUnfinishedTestFolders()
+string TestOutputHelper::detectFilterForMinusTArgument()
```

Comment:
Can you summarize what are the changes in this function? This seems unrelated to the rest

Reply:
thats for afterrun checkup of unused and empty folders. 
it will detect -t filters properly for case    <TestSuite>/<SubtestSuite>/<TestCase>
where need to look for unused and empty folders with <TestCase> names.

Reply:
I would prefer this to go into separate PR

Reply:
The main part would be merged faster then, I expect